medusa/clients/torrent/downloadstation.py

```diff
@@ -126,6 +126,10 @@ def _add_torrent_uri(self, result):
         return self._check_response()
 
     def _add_torrent_file(self, result):
+        # The API in the latest version of Download Station (3.8.16.-3566)
+        # is broken for downloading via a file, only uri's are working correct.
+        if result.url[:4].lower() in ['http', 'magn']:
```

Comment:
What about .startswith()?

Reply:
startswith() make is more complex, you have to do two test one for magnet and on for http.
This code test for http, https, and magnet links.

Reply:
You can use a tuple like: .startswith(('http', 'magn'))

Reply:
There are many ways to Rome mine is tested

Reply:
yeah your both right. But i'll go with as is. Tnx for the contribution @benjv